### ISSUE TITLE
[Suggestion] Dot monitors.

### ISSUE THREAD

**Issue by S0ZDATEL** [MightyPirates/OpenComputers#2576]
Now monitors can display text only. Of course, you can use ASCII graphics or something like that, but it is not always convenient. I suggest adding a dot monitors. They do not express the text, but separate points-pixels. They should have more resolution than "text" monitors. You can use the commands like isDot() (checks whether the monitor is dot monitor) or setDot(x, y, color) (rendering dot). You can extend the capabilities of the GPU that is now, or add special DGPU (Dot GPU). It will have a LOT of uses.

**Comment by skyem123** [comment]
We already had this suggested in various forms many times... see #1901 and #779. 😛

**Comment by Techokami** [comment]
Yeah, this is something people want, but *nobody* can agree on a proper manner to do it without dramatically impacting network performance. 

By the way, you seem new here. What's with all the feature requests in the bugtracker from you over the past few days?

**Comment by S0ZDATEL** [comment]
Yeah, I came here recently. Sorry, but I from Russia and I can't exactly understand your question. Could you rephrase it?

**Comment by Fingercomp** [comment]
You've been creating way too many feature suggestions.

Also, some of them (like this issue) are duplicates. Please check the bug tracker for a similar issue before creating a new one.

**Comment by S0ZDATEL** [comment]
Ok.

**Comment by skyem123** [comment]
Nothing's wrong with suggesting features, but if you want to talk about it in real time, I suggest using the [IRC Channel](http://webchat.esper.net/?channels=#oc).
Also, please don't make duplicate issues, there already are so many, so if an issue looks similar, it's reasonable to comment on it, extending it / debating it. Hope this helps 😄
